packages/lesswrong/lib/instanceSettings.ts

```diff
@@ -0,0 +1,113 @@
+import { initializeSetting } from './publicSettings'
+import { Meteor } from 'meteor/meteor'
+
+const getNestedProperty = function (obj, desc) {
```

Comment:
Use the version of this function in `Utils` rather than copy-pasting it

Reply:
Not an option because of complicated file-execution order. If you import from vulcan-lib in this file then you run into some circular dependency errors and vulcan lib doesn't have access to the settings classes.